arch/lkl/kernel/time.c

```diff
@@ -29,6 +29,11 @@ void calibrate_delay(void)
 {
 }
 
+void read_persistent_clock(struct timespec *ts)
```

Comment:
I was not aware (and not really awake while writing that answer). Thanks for the explanation.